var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -119,6 +128,19 @@ def install(self, spec, prefix):
             args.extend(["LIBFFTW3=%s" % fftw.ld_flags])
             args.extend(["FFTW3_INCLUDE={0}".format(spec["fftw-api"].prefix.include)])
 
+        if "+libxc" in spec:
```

Comment:
We also encourage the use of `spec.satisifies()` when checking variants to ensure the expected behavior.

From the docs:  `The difference between the two functions is that satisfies() tests whether spec constraints overlap at all, while in tests whether a spec or any of its dependencies satisfy the provided spec.` (https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints).